$NetBSD: patch-ab,v 1.1 2006/12/23 13:59:58 rillig Exp $

AM_GLIB_DEFINE_LOCALEDIR is buggy. It does not expand enough variable
references in $localedir.

--- configure.orig	2006-12-23 14:01:10.000000000 +0100
+++ configure	2006-12-23 14:47:55.000000000 +0100
@@ -23598,6 +23598,7 @@ if test "x$CATOBJEXT" = "x.mo" ; then
 else
   localedir=`eval echo "${datadir}/locale"`
 fi
+localedir="$prefix/$PKGLOCALEDIR/locale"
 prefix="$glib_save_prefix"
 exec_prefix="$glib_save_exec_prefix"
 
